src/plugins/tables/tables.js

```diff
@@ -228,16 +236,44 @@ $document.on( "submit", ".wb-tables-filter", function( event ) {
 			return;
 		}
 
+		// Verifies if filtering the same column
+		if ( $column !== $prevCol || $prevCol === -1 ) {
+			$cachedVal = "";
+		}
+		$prevCol = $column;
+
 		// Filters based on input type
 		if ( $elm.is( "select" ) ) {
 			$value = $elm.find( "option:selected" ).val();
-		} else if ( $elm.is( ":checkbox" ) ) {
+		} else if ( $elm.is( "input[type='number']" ) && $elm.val() ) {
+			var $minNum, $maxNum, $fData;
 
-			// Verifies if using same checkbox list
-			if ( $column !== $lastColumn || $lastColumn === -1 ) {
-				$cbVal = "";
+			// Retain minimum number (always the first number input)
+			if ( $cachedVal === "" ) {
+				$cachedVal = parseFloat( $elm.val() );
+			}
+			$minNum = $cachedVal;
+
+			// Maximum number is always the current selected number
+			$maxNum = parseFloat( $elm.val() );
+
+			// Generates a list of numbers (within the min and max number)
+			if ( !isNaN( $minNum ) && !isNaN( $maxNum ) ) {
+				$fData = $datatable.column( $column ).data().filter( function( obj ) {
+					var $num = $toNumber( obj.toString() );
+
+					if ( !isNaN( $num ) && $num >= $minNum && $num <= $maxNum ) {
```

Comment:
As for the other thing @GormFrank that would be here in the code, currently it must have both.

It could be `!isNaN( $num ) && ( $num >= $minNum || $num <= $maxNum )`.

Now if you wanted both the AND and OR options I need to code advance options similar to the checkboxes (already exist).